core-services/secret-mirroring/_mapping.yaml

```diff
@@ -240,3 +240,15 @@ secrets:
   to:
     namespace: ci
     name: registry-pullsecret
+- from:
+    namespace: kam
+    name: kam-user-secret
+  to:
+    namespace: ci
+    name: kam-user-secret
+- from:
+    namespace: kam
+    name: kam-user-secret
+  to:
+    namespace: ci-stg
```

Comment:
Please do not put anything into `ci-stg`, its not used anymore

Reply:
Done

@alvaroaleman I have created a pr https://github.com/openshift/release/pull/13039 to update the README.md. PTAL